internal/app/focus_manager.go

```diff
@@ -21,25 +21,42 @@ func NewFocusManager(c fyne.CanvasObject) *FocusManager {
 }
 
 // Focus focuses the given obj.
-func (f *FocusManager) Focus(obj fyne.Focusable) {
+func (f *FocusManager) Focus(obj fyne.Focusable) bool {
 	f.Lock()
 	defer f.Unlock()
 	if obj != nil {
-		if dis, ok := obj.(fyne.Disableable); ok && dis.Disabled() {
```

Comment:
Can you say why this early exit was dropped to the end?

Reply:
Because we should return `false` if the object is not part of the content but `true` if it is, regardless if we actually changed the focus.